### PR TITLE
update kubernertes version

### PR DESCRIPTION
**Pull request by jdamata**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def read(fname):
         'termcolor>=1.1.0,<2',
         'openstacksdk>=0.10.0,<1',
         'spur>=0.3.20,<1',
-        'kubernetes>=1.0.0,<2',
+        'kubernetes>=8.0.1<9',
```
Comment (mikeln) [inline setup.py]: should probably go to 9.0.x since 1.13 is current and 1.14 is coming very soon.

Comment (seeker89) [reply setup.py]: +1

**Comment by seeker89** [comment]
I should be able to take this for a spin tomorrow. Thanks for your patience !

**Comment by seeker89** [comment]
I'll need to do a pass and update some other dependencies, so let's get this in for now. Thanks !

**Pull request merged**